changelog_unreleased/graphql/12608.md

```diff
@@ -0,0 +1,84 @@
+#### Fix description markdown formatting (#12608 by @chimurai)
+
+<!-- prettier-ignore -->
+```graphql
+# Input
+""" Customer """
+type Person {
+  name: String
+}
+
+# Prettier 2.6.2
+"""
+ Customer
+"""
```

Comment:
Why don't we try print the comment inline, if:

- can fit
- was inline originally